src/App.tsx

```diff
@@ -1,28 +1,69 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { FilterStatusType, Todo } from './types/Todo';
+import * as tadoService from './api/todos';
+import classNames from 'classnames';
+import { Footer } from './component/Footer';
+import { Error } from './component/Error';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+
+function todoInUse(todo: Todo[], filter: FilterStatusType) {
```

Comment:
```suggestion
function filterTodos(todos: Todo[], filter: FilterStatusType) {
```
- The function name should start from the verb
- Move to utils/helpers file/folder
- Probably todos should be plural as you accept not only one todo